### PR TITLE
i18n(ko-KR): update `configuration.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description

- update `configuration.mdx`
- #1729

### CODE REVIEW

File: docs/src/content/docs/ko/reference/configuration.mdx [hunk 1]
```diff
@@ -447,6 +447,10 @@ starlight({
 });
 ```
 
+`head`의 항목은 HTML 요소로 직접 변환되며 Astro의 [스크립트](https://docs.astro.build/ko/guides/client-side-scripts/#스크립트-처리)나 [스타일](https://docs.astro.build/ko/guides/styling/#astro에서-스타일링하기) 처리를 거치지 않습니다.
+
+스크립트, 스타일 또는 이미지와 같은 로컬 자산을 가져와야 하는 경우 [Head 컴포넌트를 재정의하세요](/ko/guides/overriding-components/#내장-컴포넌트-재사용).
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/reference/configuration.mdx:452]:
```suggestion
스크립트, 스타일 또는 이미지와 같은 로컬 자산을 가져와야 하는 경우 [Head 컴포넌트를 재정의하세요](/ko/guides/overriding-components/#내장-컴포넌트-재사용).
```

Tiny detail to match the original version

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view), left a tiny suggestion for an empty line, but except for that, everything looks perfect to me.

**Comment by jsparkdev** [comment]
thank you!

**Review by HiDeoo — APPROVED** [review]
Thanks for the update 🚀

**Pull request merged**